<launch>

    <node pkg="uwb_manager" type="UwbManager_Node" name="UwbManager" output="screen" >
		<param name="dev" type="string" value="/dev/ttyACM0" />
    </node>

</launch>
